tests/integration/cqlengine/statements/test_base_statement.py

```diff
@@ -101,6 +101,7 @@ def test_insert_statement_execute(self):
         self.assertEqual(TestQueryUpdateModel.objects.count(), 0)
 
     @greaterthanorequalcass3_10
+    @lessthandse69
```

Comment:
Wait, why does this test... fail on DSE 6.9?  Did we remove support for LIKE or something?

Reply:
Oh, it's because of the removal of SASI.. I see.

Can we replace this with a new SAI test then?

Reply:
Unfortunately not the `LIKE` operator.

> SAI does not supports the following query operators for tables with SAI indexes:
> - Strings or Numerics: LIKE, OR, IN

Documentation [link](https://docs.datastax.com/en/cql/dse-6.9/develop/indexing/sai/sai-query.html).

Reply:
The [CEP](https://cwiki.apache.org/confluence/display/CASSANDRA/CEP-7%3A+Storage+Attached+Index) has it as a "Phase 2" thing which puts it in the same category as OR queries which were discussed at some length for the Java driver (see [CASSANDRA-19931](https://issues.apache.org/jira/browse/CASSANDRA-19931)).  Excluding for 6.9.0 and up seems like the right answer until we get a better answer from SAI.